cpp/memilio/epidemiology/state_age_function.h

```diff
@@ -74,13 +82,19 @@ struct StateAgeFunction {
      * 
      * @param[in] init_parameter Specifies the initial function parameter of the function.
      */
-    StateAgeFunction(ScalarType init_parameter)
+    StateAgeFunction(ScalarType init_parameter, ScalarType init_location = 0, ScalarType init_scale = 1)
         : m_parameter{init_parameter}
```

Comment:
Should we rename `m_parameter` to `m_distrib_parameter` or something like that? When there are multiple parameters the name `m_parameter` is not very descriptive.

Reply:
Yes, I like this. Done.